src/main/js/isd.js

```diff
@@ -59,22 +59,90 @@
 
         };
 
-        /* process regions */
+        /* Filter body contents - Only process what we need within the offset and discard regions not applicable to the content */
+        var body = {};
+        var activeRegions = {};
+
+        /* gather any regions that might have showBackground="always" and show a background */
+        initialShowBackground = tt.head.styling.initials[imscStyles.byName.showBackground.qname];
+        initialbackgroundColor = tt.head.styling.initials[imscStyles.byName.backgroundColor.qname];
+        initialbackgroundImage = tt.head.styling.initials[imscStyles.byName.backgroundImage.qname];
```

Comment:
```suggestion
```

`backgroundImage` does not apply to `region`

Reply:
Ah, [`tts:backgroundImage`](https://www.w3.org/TR/ttml2/#style-attribute-backgroundImage) _does_ apply to `region`. However I was overzealous here: it is not currently permitted in IMSC, and the alternative `smpte:backgroundImage` is, as you say, not applicable to `region`. I'll remove this.

I also see that IMSC's [`.12.1.1 Presented Region`](https://www.w3.org/TR/ttml-imsc1.2/#presented-region) defines the semantics that are relevant here in a concise way, helpfully.

Reply:
Fixed in 9c3dc7b